### DISCUSSION TITLE
Problem when converting Custom Tkinter GUI to .exe

### DISCUSSION THREAD

**Discussion by SmartCoder12345** [TomSchimansky/CustomTkinter#2323] [Q&A] [answered]
I have created a Custom TKinter GUI. I converted it into .exe using the code of your documentation. The Problem is that it is only recognizing the modules of My main.py and not of custom tkinter's. I hope You understood my problem. Pls give me the solution 🥺

**Comment by Akascape** [comment, marked as answer]
@SmartCoder12345 I have a full guide/tutorial on this topic, here is the link: https://www.akascape.com/coding/best-pyinstaller-guide

Also check this thread: #939
